composer.json

```diff
@@ -42,7 +35,8 @@
         "symfony/yaml": "^4.1",
         "nelmio/api-doc-bundle": "^3.2",
         "symfony/twig-bundle": "^4.1",
-        "symfony/asset": "^4.1"
+        "symfony/asset": "^4.1",
+        "adshares/php-ads-client": "^1.0@dev"
```

Comment:
Zmień na `dev-master` lub `^1.0`

Reply:
I'll change it manually because `composer require adshares/php-ads-client` installs `1.0@dev` by default. How it works from Packageist point of view?

Reply:
Done

Reply:
By default composer install the newest commit from the `master` branch (`dev-master`). This is mapped to a "real" version in the `composer.json` file:
```
"extra": {
        "branch-alias": {
            "dev-master": "1.0.x-dev"
        }
    },
```
So if you have unreleased commits, the composer will install `1.0@dev` by default.